controllers/usersignup/usersignup_controller.go

```diff
@@ -574,12 +579,17 @@ func (r *Reconciler) sendDeactivatingNotification(logger logr.Logger, userSignup
 	return nil
 }
 
-func (r *Reconciler) sendDeactivatedNotification(logger logr.Logger, userSignup *toolchainv1alpha1.UserSignup) error {
+func (r *Reconciler) sendDeactivatedNotification(logger logr.Logger, config toolchainconfig.ToolchainConfig, userSignup *toolchainv1alpha1.UserSignup) error {
+	keysAndVals := map[string]string{
+		"RegistrationURL": config.RegistrationService().RegistrationServiceURL(),
```

Comment:
And here.

Reply:
Done 39b6f7c058b917f01984d2f7b3268aa562d68e40